Mods/Pet_Default_v5_Dark.theme

```diff
@@ -1,20 +1,24 @@
 @description Default v5 Dark Theme
-@version 1.0
+@version 1.1
 @changelog
   - Updated to v5.22 (added preview)
+v1.1
```

Comment:
This un-indented line (the `v1.1`) closes the changelog tag (= the next three lines aren't read). Also you don't need to keep the changelogs of older versions as ReaPack remembers those for you.
```
@version 1.1
@changelog
  - MIDI Editor Colormap reworked
  - Changes in the ME Tracklist
  - Separators added 
```